.travis.yml

```diff
@@ -1,3 +1,11 @@
+---
+sudo: false
 language: ruby
+cache: bundler
 rvm:
-  - 1.9.3
```

Comment:
it could be some people still use 1.9.3 (?)

Reply:
Well, it ended its life and is not supported by the ruby-core team any longer and does not receive any fixes. I don't see a reason to release a new version supporting dead ruby versions. If someone still uses it, he probably doesn't upgrade anything including dependent gems ;) 

The only maintained versions are 2.4 (until end of this month), 2.5, 2.6, 2.7.

Reply:
Good point. I'm for the removal from CI also then.